### DISCUSSION TITLE
Can someone help me with this problem!!! I have downloaded games in PlayCover, but the games just won't open. How to solve thisss!

### DISCUSSION THREAD

**Discussion by Joshwangder** [PlayCover/PlayCover#1457] [Q&A]
The game keep on crashing, I tried redownloaded. It didn't work, I also tried restarting my computer. DIdn't work. PLZ help me solve this queston!!!!!!!!!!!!!

**Comment by TheMoonThatRises** [comment]
Can you provide more information? Also please create an issue next time to report problems.
